CONTRIBUTING.md

```diff
@@ -23,7 +49,24 @@ $ bazel test //...
 
 This command runs all the tests.
 
-## Submit a pull request
+## Squashing your commits
+
+To keep a clean track of what is being released, every feature must contain only one commit when merged.
+The commit message of the squashed commit is very important, since it will be used to keep track of the features in a release.
+
+The convention is [#issue - description], for the example issue "Map message content" with number 216, the squashed commit message would be
+
+`[#216 - Map message content]`
```

Comment:
I thought we had settled on `- [#123] Do the important thing`? 🤔

Reply:
👍 Yes, it's way better. Will change it